infra.bs

```diff
@@ -799,6 +799,14 @@ out-of-bounds, except when used with <a for=list>exists</a>.
 <p>To <dfn export for=list>append</dfn> to a <a>list</a> that is not an <a>ordered set</a> is to
 add the given <a for=list>item</a> to the end of the list.
 
+<p>To <dfn export for=list>extend</dfn> a <a>list</a> |A| with a <a>list</a> |B|,
+<a for=list>for each</a> |item| of |B|, <a for=list>append</a> |item| to |A|.
+
+<p class=example id=example-list-extend>To <a for=list>extend</a> « "<code>Erin Gilbert</code>",
+"<code>Abby Yates</code>" » with « "<code>Jillian Holtzmann</code>", "<code>Patty Tolan</code>" »
+gives « "<code>Erin Gilbert</code>", "<code>Abby Yates</code>", "<code>Jillian Holtzmann</code>",
```

Comment:
I think "gives" makes it a big unclear that this mutates in place. I'm OK keeping it as-is, since the normative text is right above and is pretty clear. But you could instead do an example that uses named variables and then asserts that the size is 4 after the extend operation.